packages/Path-View-Core.package/SClassDiagram.class/instance/boundingBoxOfClassMorphs.st

```diff
@@ -0,0 +1,10 @@
+geometry
+boundingBoxOfClassMorphs
+	"calculate the bounding box of all class morph in the view"
+
+	| emptyRectangle |
+	emptyRectangle := 0@0 corner: 0@0.
+	^ self classMorphs 
```

Comment:
we should decide whether we want to access instance variables with `self classMorphs` or directly with `classMorphs`.